### ISSUE TITLE
Why are paths relative to Downloads required on the Firefox version?

### ISSUE THREAD

**Issue by SteegHobland** [edjroot/webext-save-to-folder#5]
It is said to be Chrome's fault that paths relative to the Downloads folder are required. Why then is this limitation also in the Firefox version? Does Firefox limit you to relative paths as Chrome does?

**Comment by edjroot** [comment]
Sorry for the super late response! I always miss these notifications...

TL;DR: Yes, Firefox and Chrome have the same behavior, and I need to update the README.

As far as I know (I may be wrong), very generally speaking, Mozilla's aim with WebExtensions is to provide identical behavior to Chrome's extensions for existing APIs, so the default answer for this kind of question would be "yes, the behavior is the same, or else it's a bug".

Now, answering your specific question, this is what [MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/downloads/download) has to say:

> Absolute paths, empty paths, and paths containing back-references (`../`) will cause an error. If omitted, this value will default to the filename already given to the download file, and a location immediately inside the downloads directory.

Finally, the reason why only Chrome is mentioned is that I initially developed this extension for Chrome and haven't properly updated the README since then. Sorry for that! I'm going to do that now if procrastination doesn't suddenly slap me.

I'm going to close this issue since it was a simple yes/no question, but don't hesitate to ask more questions. I'll try not to miss them. :p

Hope I was helpful!
